README.md

```diff
@@ -29,20 +29,25 @@ To learn more about Spacemesh head over to [https://spacemesh.io](https://spacem
 To learn more about the Spacemesh protocol [watch this video](https://www.youtube.com/watch?v=jvtHFOlA1GI).
 
 ### Motivation
+
 Spacemesh is designed to create a decentralized blockchain smart contracts computer and a cryptocurrency that is formed by connecting the home PCs of people from around the world into one virtual computer without incurring massive energy waste and mining pools issues that are inherent in other blockchain computers, and provide a provably-secure and incentive-compatible smart contracts execution environment.
 
 Spacemesh is designed to be ASIC-resistant and in a way that doesn’t give an unfair advantage to rich parties who can afford setting up dedicated computers on the network. We achieve this by using a novel consensus protocol and optimize the software to be most effectively be used on home PCs that are also used for interactive apps.
 
 ### What is this good for?
+
 Provide dapp and app developers with a robust way to add value exchange and other value related features to their apps at scale. Our goal is to create a truly decentralized cryptocurrency that fulfills the original vision behind bitcoin to become a secure trustless store of value as well as a transactional currency with extremely low transaction fees.
 
 ### Target Users
+
 go-spacemesh is designed to be installed and operated on users' home PCs to form one decentralized computer. It is going to be distributed in the Spacemesh App but people can also build and run it from source code.
 
 ### Project Status
-We are working hard towards our first major milestone - a public permissionless testnet running the Spacemesh consensus protocol.
+
+We are working hard towards our first major milestone - a public permission-less testnet running the Spacemesh consensus protocol.
```

Comment:
```suggestion
We are working hard towards our first major milestone - a public permissionless testnet running the Spacemesh consensus protocol.
```

Reply:
add it to dictionary without hyphen. it's a word if you're working with blockchains.
adding the hyphen sounds like we just use permissions less than others.
spacemesh is a truly permissionless distributed system.